server/utils/projects.js

```diff
@@ -0,0 +1,29 @@
+// Projects Validations
+const { isValidStr, isValidObj, isValidArray } = require('./index');
+const isValidProjectName = (projectName) => {
+	const projectName = isValidStr(projectName, 'project name', 'min', 3);
```

Comment:
i think we need more validations here, such as no special characters. only alphanumeric